### DISCUSSION TITLE
[ Icon Suggestion ] Social Medias

### DISCUSSION THREAD

**Discussion by Pyr33x** [tailwindlabs/heroicons#924] [Ideas]
Heya!
There is many many icons in heroicons but where are the social media icons? ( like twitter, instagram, github etc.. )
You guys gotta think about this

**Comment by zefir-git** [comment]
You could use https://simpleicons.org — Simple SVG icons of thousands of brands (free and open-source https://github.com/simple-icons/simple-icons). The website also tells you what their primary brand colour is and has a link to logo usage guidelines if present.

I personally don‘t think heroicons needs brand icons since these are usually already provided by the brand as SVG and there isn‘t much you can adapt them to make them fit the rest of the heroicons, i.e. you can‘t edit a company‘s logo just so that it matches your website (at least afaik). And when the brand doesn‘t provide an SVG on their "Brand usage" or similar page, you can often find it online. One good place (if you don‘t want to use simpleicons.org) is Wikipedia. It often has an SVG logo of the brand when you open its page (sometimes it's a PNG rendering and there's a link labelled something like "Show original SVG"). And many other websites also provide brand logos.

> **Note**: To use simpleicons.org with text colour (i.e. `text-blue-500` in TW, etc.) you need to add `fill="currentColor"` to the `<svg>` tag.

**Reply by Pyr33x** [reply]
Gotcha, thanks buddy!
